strongbox-rest-client/src/main/java/org/carlspring/strongbox/client/RestClient.java

```diff
@@ -1,5 +1,23 @@
 package org.carlspring.strongbox.client;
 
+import static org.carlspring.strongbox.db.schema.Properties.PASSWORD;
```

Comment:
@liooh could you please don't override all the imports.

Reply:
I went back to the previous state, just adding the statics imports for that file